go/vt/vtctl/workflow/traffic_switcher.go

```diff
@@ -487,23 +487,29 @@ func (ts *trafficSwitcher) dropParticipatingTablesFromKeyspace(ctx context.Conte
 func (ts *trafficSwitcher) removeSourceTables(ctx context.Context, removalType TableRemovalType) error {
 	err := ts.ForAllSources(func(source *MigrationSource) error {
 		for _, tableName := range ts.Tables() {
-			query := fmt.Sprintf("drop table %s.%s",
-				sqlescape.EscapeID(sqlescape.UnescapeID(source.GetPrimary().DbName())),
-				sqlescape.EscapeID(sqlescape.UnescapeID(tableName)))
+			primaryDbName, err := sqlescape.EnsureEscaped(source.GetPrimary().DbName())
+			if err != nil {
+				return err
+			}
+			tableName, err := sqlescape.EnsureEscaped(tableName)
+			if err != nil {
+				return err
+			}
+
+			query := fmt.Sprintf("drop table %s.%s", primaryDbName, tableName)
 			if removalType == DropTable {
 				ts.Logger().Infof("%s: Dropping table %s.%s\n",
 					source.GetPrimary().String(), source.GetPrimary().DbName(), tableName)
 			} else {
-				renameName := getRenameFileName(tableName)
+				renameName, err := sqlescape.EnsureEscaped(getRenameFileName(tableName))
```

Comment:
This is wrong. It's causing CI to fail, because it doesn't first unescape `tableName`, then do `getRenameFileName` and then escapes it again. So you get an error like:

```
 code = Unknown desc = UnescapeID err: no outer backticks found in the identifier '_`customer2`_old'
```

because of the broken table name.

Reply:
made the changes, CI is also passing